Plugins/ActSVG/include/Acts/Plugins/ActSVG/IndexedSurfacesSvgConverter.hpp

```diff
@@ -29,10 +29,12 @@
 namespace Acts::Svg {
 
 using ProtoSurface = actsvg::proto::surface<std::vector<Vector3>>;
+using ProtoSurfaces = std::vector<ProtoSurface>;
```

Comment:
I was wondering, if we could use this type alias also later on, eg in l.63
```
  // The return surfaces
  std::vector<ProtoSurface> pSurfaces;
```

same for the other aliases.